src/main/kotlin/no/nav/tilleggsstonader/sak/journalføring/JournalføringHelper.kt

```diff
@@ -71,4 +76,11 @@ object JournalføringHelper {
             }
         },
     )
+
+    fun utledNyAvsender(nyAvsender: JournalføringRequest.NyAvsender?, bruker: Bruker?): AvsenderMottaker? =
+        when (nyAvsender?.erBruker) {
+            null -> null
+            true -> AvsenderMottaker(id = nyAvsender.personIdent, idType = BrukerIdType.FNR, navn = nyAvsender.navn ?: error("Mangler navn på bruker"))
```

Comment:
Hvis man sender inn true, burde ikke verdiene fra bruker brukes då? 
```suggestion
            true -> AvsenderMottaker(id = bruker.personIdent, idType = BrukerIdType.FNR, navn = bruker.navn ?: error("Mangler navn på bruker"))
```

Reply:
Usikker, kopiert rett fra 😅  Men burde jo egentlig det, uten at jeg helt skjønner hvorfor det er gjort sånn her